kotlin/kotlintest/README.md

```diff
@@ -0,0 +1,25 @@
+# Java Truth
+
+Minimal setup with Kotlin and kotlintest to get you started
+
+## Setup
+
+    git clone https://github.com/swkBerlin/kata-bootstraps
+    cd kotlin/kotlintest
+    gradle idea
```

Comment:
would you mind including a gradle wrapper? i'm on a machine right now without gradle installed...